inspirehep/modules/migrator/tasks.py

```diff
@@ -288,9 +289,27 @@ def create_record(record):
     return json
 
 
+def attach_documents_and_figures(record):
```

Comment:
this function is probably (or can be adapted) to be the same as the one in the other pr: https://github.com/inspirehep/inspire-next/pull/2760/files#diff-3cf20a9f11cd40bad5e31856ef439299R94

We should put that function in the records module and reuse it from there, probably renamed.

Reply:
That means though basing one PR in the other... so as this one is shorter and probably will get merged first, I'd use this one as the base.